cypress/features/regression/test/flatTableIniFrame.feature

```diff
@@ -5,31 +5,35 @@ Feature: FlatTable component
     Given I open basic Test "Flat Table" component page
 
   @positive
-  Scenario: FlatTable has sticky row header
+  Scenario: FlatTable has sticky row
+    # commented because of BDD default scenario Given - When - Then
+    # I open "test-flat-table" component page basic
     When I check hasHeaderRow checkbox
-    Then FlatTable has sticky row header
+    Then FlatTable has sticky row
 
   @positive
   Scenario: FlatTable has sticky header
+    # commented because of BDD default scenario Given - When - Then
+    # I open "test-flat-table" component page basic
     When I check hasStickyHead checkbox
     Then FlatTable has sticky header
 
   @positive
-  Scenario: Check that FlatTable has nine rows
+  Scenario: FlatTable has nine rows
     # commented because of BDD default scenario Given - When - Then
-    # When I open "Flattable" component page
+    # I open "test-flat-table" component page basic
     Then FlatTable has nine rows
 
   @positive
-  Scenario: Check that FlatTable has seven columns
+  Scenario: FlatTable has seven columns
     # commented because of BDD default scenario Given - When - Then
-    # When I open "Flattable" component page
+    # I open "test-flat-table" component page basic
```

Comment:
put the word `When` in front of `I open...`

Reply:
fixed